assessments/guide/security-reviewer.md

```diff
@@ -53,14 +53,37 @@ guidelines for conflict of interest situations are important to prevent:
 - SIG-Security chairs and assessment leads could intentionally or unintentionally limit the participation of an individual unfairly by asserting conflict of interest
 - Security reviews being stalled while groups belabor on who should be allowed to participate
 
-The conflicts of interest lie on a spectrum, and are classified into hard and soft conflicts.
-Hard conflicts makes a reviewer ineligible to review a project.
-A soft conflict allows a reviewer to review a project, but not as a [project lead](./project-lead.md).
-A reviewer with a soft conflict is required to document the nature of the conflict in the
-security assessment documentation. It is the responsibility of the security reviewer
-to make known his/her own conflict of interests.
-
-For each project, 2 SIG-Security chairs must sign off on the conflicts presented to them that the assessment lead has no conflicts, and reviewers have no hard conflicts. Reasons for accepting and rejecting conflicts should be documented.
+The conflicts of interest lie on a spectrum, and are classified into hard and soft conflicts:
+* Hard conflicts make a reviewer ineligible to review a project.
+* A soft conflict allows a reviewer to review a project, but not as a [project lead](./project-lead.md).
+* Soft conflicts are expected for some reviewers, and therefore are fine as long
+  as the review team includes enough different perspectives to ensure a
+  balanced assessment.
+
+A reviewer with a any conflict or potential conflect is required to document the nature of the conflict when volunteering as a reviewer, or at any time as they
+become aware of a potential conflict. It is the responsibility of the security 
+reviewer to make known his/her own conflict of interests and ensure that
+their declaration has been reviewed and approved by lead security reviewer.
+
+The lead security reviewer has the responsibility of ensuring a balanced review,
+and as part of that before kicking off the assessment must:
+* check that all reviewers have conflict-of-interest declarations
+* make their declaration of any potential conflict-of-interest (or lack thereof)
+* ask clarifying questions, if needed
+* assert that:
+  1. they have reviewed all conflict-of-interest declarations;
+  2. there are no hard-conflicts;
+  4. they believe that the review team is able to provide a balanced and fair    assessment
+
+Update the above assertion if a new conflict-of-interest becomes known during
+the course of the assessment.
+
+The Security Assessment Facilitator or a SIG-Security chair must review the 
+Lead Security Reviewer conflict-of-interest assertion.
+
+If there any conflicts are presented, then 2 SIG-Security chairs must approve
```

Comment:
This is for soft conflicts, I assume.  Can hard conflicts ever be waived?  Can a conflicted person still participate unofficially?

Reply:
It's hard to imagine that we would waive a hard conflict, I'm open to writing down that we would disallow chair approval of such.

I think we say elsewhere that any SIG member (or community member?) can participate... would it be helpful to make it explicit here?  I don't want to imply that there is some kind of "unofficial reviewer" role... just cuz that feels complicated.  Maybe someone can suggest words :)